src/jsonapi.js

```diff
@@ -49,34 +48,43 @@ export const uploadFile = (file, {
   console.warn('uploadFile has been deprecated and will no longer be supported by redux-json-api https://github.com/dixieio/redux-json-api/issues/2');
 
   return (dispatch, getState) => {
-    const accessToken = getState().api.endpoint.accessToken;
+    const axiosConfig = getState().api.endpoint.axiosConfig;
     const path = [companyId, fileableType, fileableId].filter(o => !!o).join('/');
-    const url = `${__API_HOST__}/upload/${path}?access_token=${accessToken}`;
+    const endpoint = `${__API_HOST__}/upload/${path}`;
 
     const data = new FormData;
     data.append('file', file);
 
     const options = {
+      ...axiosConfig,
       method: 'POST',
       body: data
     };
 
-    return fetch(url, options)
-      .then(res => {
-        if (res.status >= 200 && res.status < 300) {
-          if (jsonContentTypes.some(contentType => res.headers.get('Content-Type').indexOf(contentType) > -1)) {
-            return res.json();
-          }
-
-          return res;
-        }
-
-        const e = new Error(res.statusText);
-        e.response = res;
-        throw e;
-      })
+    // return fetch(url, options)
+    //   .then(res => {
+    //     if (res.status >= 200 && res.status < 300) {
+    //       if (jsonContentTypes.indexOf(res.headers.get('Content-Type')) > -1) {
+    //         return res.json();
+    //       }
+    //
+    //       return res;
+    //     }
+    //
+    //     const e = new Error(res.statusText);
+    //     e.response = res;
+    //     throw e;
+    //   })
+    //   .then(json => {
+    //     onSuccess(json);
+    //   })
+    //   .catch(error => {
+    //     onError(error);
+    //   });
+
+    return apiRequest(endpoint, options)
```

Comment:
`options` is provided as second argument, which is supposed to be `accessToken`. It does appear that `accessToken` is no longer used, though.